src/lexer.coffee

```diff
@@ -946,19 +947,19 @@ exports.Lexer = class Lexer
 
   # Same as `token`, except this just returns the token without adding it
   # to the results.
-  makeToken: (tag, value, offsetInChunk = 0, length = value.length) ->
+  makeToken: (tag, value, offsetInChunk = 0, length = value.length, origin) ->
```

Comment:
@GeoffreyBooth ah I see ya once this is merged I can merge into `preserve-string-literal` and convert to an option arg that gets passed along by `token()`